src/collections/_documentation/platforms/native/index.md

```diff
@@ -194,6 +194,33 @@ initialization:
   sentry_options_set_release(options, "my-unique-release-1.0.0");
 ```
 
+### Release Health
+
+Monitor the [health of releases](/workflow/releases/health/) by observing user adoption, usage of the application, percentage of [crashes](/workflow/releases/health/#crash), and [session data](/workflow/releases/health/#session). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
+To benefit from the health data you must use at least version 0.4.0 of the Native SDK.
+The collection of release health metrics is enabled by default unless specifically disabled in the initialization options of the SDK.
+
+```c
+sentry_options_t *options = sentry_options_new();
+// Session tracking is enabled by default. To disable:
+sentry_options_set_auto_session_tracking(options, 0);
+sentry_init(options);
+```
+
+The SDK automatically starts a new session when it is initialized via `sentry_init`, and will end that session automatically when the SDK is shut down via `sentry_shutdown`, or explicitly by calling `sentry_end_session`.
+The SDK currently only tracks one concurrent session and will also end the running session when using `sentry_start_session`.
```

Comment:
No reason to start a new line here (is this from the conversion, perhaps?)

Minor edit (use "only" immediately adjacent to the phrase it is modifying): The SDK currently tracks only one concurrent session and will end the running session when using `sentry_start_session`.

Reply:
Those line breaks are silent, they will not show up in the rendered output.